resume_sandbox/auth.py

```diff
@@ -47,42 +47,42 @@ def login():
 		user = db.execute(
 			'SELECT * FROM user WHERE username = ?', (username,)
 		).fetchone()
-		
+
 		if user is None:
 			error = 'Incorrect username.'
 		elif not check_password_hash(user['password'], password):
 			error = 'Incorrect password.'
-			
+
 		if error is None:
 			session.clear()
 			session['user_id'] = user['id']
-			return redirect(url_for('index'))
-		
+			return redirect(url_for('home'))
```

Comment:
Not sure if it's due to how my venv is setup for flask, but I had to change 'home' to 'sandbox.home' to login successfully.